crates/holochain/src/conductor/cell.rs

```diff
@@ -570,61 +570,51 @@ impl Cell {
 */
 
 impl holochain_p2p::event::HcP2pHandler for Cell {
```

Comment:
Sorry for the diff here, I maybe shouldn't have done this but there was a "the methods are in a different order to the trait" warning here. That makes it slow for me to find things and I was in and out of this file again and again.